crates/aquascope_front/src/plugin.rs

```diff
@@ -220,10 +221,10 @@ impl<A: AquascopeAnalysis> rustc_driver::Callbacks for AquascopeCallbacks<A> {
     config.override_queries = Some(borrowck_facts::override_queries);
   }
 
-  fn after_expansion<'tcx>(
+  fn after_analysis<'tcx>(
```

Comment:
FYI, from what I can tell, this is *not* called if the analysis fails. It should be `after_expansion` because we bypass the borrow checker.